tool/listpkgs/listpkgs.go

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+// listpkgs prints the import paths that match the Go package patterns
+// given on the command line and conditionally filters them in various ways.
+package main
+
+import (
+	"bufio"
+	"cmp"
+	"flag"
+	"fmt"
+	"go/build/constraint"
+	"log"
+	"os"
+	"runtime"
+	"slices"
+	"strings"
+
+	"golang.org/x/tools/go/packages"
+)
+
+var (
+	ignore3p    = flag.Bool("ignore-3p", false, "ignore third-party packages forked/vendored into Tailscale")
```

Comment:
Suggestion: maybe  `--include-3p` with default false? Since that's how it's used in the test

Reply:
Well, all the other flags purely remove things. No flag adds things in. It kinda seems like a double negative to invert it.

Reply:
fair